src/qibocal/protocols/coherence/t1.py

```diff
@@ -28,9 +28,7 @@ class T1Parameters(t1_signal.T1SignalParameters):
 class T1Results(t1_signal.T1SignalResults):
     """T1 outputs."""
 
-    chi2: Optional[dict[QubitId, tuple[float, Optional[float]]]] = field(
-        default_factory=dict
-    )
+    chi2: Optional[dict[QubitId, list[float, float]]] = field(default_factory=dict)
```

Comment:
I have not found any example of `list[float, float]`, could you explaining the what does it means ? I think `list[float]` is enough

Reply:
Yes, indeed it should be `list[float]`, thanks
```suggestion
    chi2: Optional[dict[QubitId, list[float]]] = field(default_factory=dict)
```